src/helpers/terminal_helpers.sh

```diff
@@ -136,3 +136,11 @@ function run_command {
   print_command "$cmd"
   eval "$cmd" 2>&1
 }
+
+
+# Run a command, prints command and output
+function run_command_outside_git {
```

Comment:
How about naming this `run_command` and rename the other functions to `run_git_command` and `print_git_command`. Throw the rename into another PR that should go through quickly.